app/src/main/java/org/p2p/wallet/common/analytics/constants/EventNames.kt

```diff
@@ -95,6 +95,15 @@ object EventNames {
     const val SEND_ACTION_BUTTON_CLICKED = "Action_Button_Send"
     const val SEND_IS_USERNAME = "Send_Username"
 
+    // new send
+    const val NEW_SEND_RECIPIENT_VIEWED = "Sendnew_Recipient_Screen"
```

Comment:
i suggest extracting these fields to NewSendAnalytics, cuz they are used only there